### PR TITLE
Use the xlink prefix

### PR DESCRIPTION
**Pull request by kensanata**
This matches the sentence on line 71

### Description

The text says:

> This is done by putting the namespace prefix and a colon before the parameter name as shown on the <script> element in the example above.

For this to be true, the script element in the example should use the xlink prefix even though these days it is not strictly necessary.

Since `href` is allowed on elements without the `xlink` namespace, perhaps a different example could be used? This fix is the smallest fix to bring example and text into alignment.

### Motivation

Align example and explanation.

### Additional details

none

### Related issues and pull requests

none

### CODE REVIEW

File: files/en-us/web/svg/namespaces_crash_course/index.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ By default, parameters don't have a namespace at all. They are only known to be
 <svg
   xmlns="http://www.w3.org/2000/svg"
   xmlns:xlink="http://www.w3.org/1999/xlink">
-  <script href="cool-script.js" type="text/ecmascript" />
+  <script xlink:href="cool-script.js" type="text/ecmascript" />
```
Comment (Josh-Cena) [inline files/en-us/web/svg/namespaces_crash_course/index.md:65]:
```suggestion
  <script xlink:href="cool-script.js" type="text/javascript" />
```

**Review by Josh-Cena — COMMENTED** [review]
For other reviewers to consider: is it possible to find another attribute example, since `xlink:href` is not needed now?

**Comment by kensanata** [comment]
@Josh-Cena In a conversation with @Ryuno-Ki we wondered about using Inkscape as an example.

The following is what you get when saving an empty file in Inkscape.

```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!-- Created with Inkscape (http://www.inkscape.org/) -->

<svg
   width="210mm"
   height="297mm"
   viewBox="0 0 210 297"
   version="1.1"
   id="svg5"
   inkscape:version="1.2.2 (b0a8486541, 2022-12-01)"
   sodipodi:docname="drawing.svg"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <sodipodi:namedview
     id="namedview7"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:showpageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     inkscape:deskcolor="#d1d1d1"
     inkscape:document-units="mm"
     showgrid="false"
     inkscape:zoom="0.702883"
     inkscape:cx="396.93662"
     inkscape:cy="562.68255"
     inkscape:window-width="1920"
     inkscape:window-height="1011"
     inkscape:window-x="0"
     inkscape:window-y="0"
     inkscape:window-maximized="1"
     inkscape:current-layer="layer1" />
  <defs
     id="defs2" />
  <g
     inkscape:label="Layer 1"
     inkscape:groupmode="layer"
     id="layer1" />
</svg>
```

I could use the above and rewrite the entire "Declaring namespace prefixes" section to refer to this example?

**Comment by Ryuno-Ki** [comment]
@Josh-Cena Thank you for picking this up. @kensanata reached out to me on Mastodon and we chatted about the PR.
Due to sickness and childcare my time on the laptop is currently limited. Chatting on mobile is easier to do.

@kensanata You have merge conflicts here.

**Comment by kensanata** [comment]
The question remains: proceed with the minimal solution in this merge request or rewrite a bigger chunk to use the Inkscape example?

**Comment by Ryuno-Ki** [comment]
Right, on namespace. I agree that the example has room for improvement. Originally I thought about suggesting to have a fix here and propose an improvement in another PR. Given that it is currently a one-line change, I'm inclined to have the improvement in this PR. (The PR subject would then be updated as well to reflect the commit message on merge).

So our conversation went to which namespace would be a good choice. MathML came to my mind first (because I have studied the subject). I remembered having edited a SVG the other day that was created in Inkscape. Adobe products would produce also SVGs with additional namespace but I would like to promote free software if given the chance.

The German Wikipedia has a list of xml namespaces:

https://de.wikipedia.org/wiki/Liste_der_XML-Namensr%C3%A4ume

The XML namespace itself is also documented:

https://de.wikipedia.org/wiki/Namensraum_(XML)#Das_Pr%C3%A4fix_xml

Looking at these, I could imagine that the `xml:lang` could work for multi lingual descriptions.

In fact, the specification gives an example for the `<title>` element using the `lang` attribute:

https://www.w3.org/TR/SVG2/struct.html#DescriptionAndTitleElements

(see also the listed issue about lack of known implementations that would support this).

Finally, we could go with the example provided in the spec here:

https://www.w3.org/TR/SVG2/struct.html#ForeignNamespaces

It wouldn't be too useful as the namespace is likely not supported by any implementation.

**Comment by kensanata** [comment]
I suspect that the Inkscape file would be something that people see often and wonder a lot. The title tag has the drawback that people are both unlikely to run into and unlikely to need themselves. If you want to go with a different example that doesn't refer to a product like Inkscape, a simple option would be writing a SVG file that contains XHTML elements.

The two options I'm willing to invest time into would therefore be:

1. empty Inkscape file explained
2. simple SVG file containing an XHTML form

**Comment by Ryuno-Ki** [comment]
If I would decide, I'd go with option 2 and a button as non-SVG element. This could be an example of an interactive SVG.

Similar to [Image Map](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/map)s.

**Comment by kensanata** [comment]
The following SVG fragment uses the XHTML namespace to add a button which calls a script that changes some attributes on the text element, making it jump around.

```xml
<svg version="1.1"
     width="300" height="200"
     xmlns="http://www.w3.org/2000/svg"
     xmlns:xhtml="http://www.w3.org/1999/xhtml">
  <rect width="100%" height="100%" fill="none" stroke="black" stroke-width="2px"/>
  <text id="message" x="150" y="100" text-anchor="middle" >Press the button!</text>
  <foreignObject x="10" y="10" width="80" height="50">
    <xhtml:button id="btn" onclick="jump()">Jump!</xhtml:button>
  </foreignObject>
  <script type="application/javascript"><![CDATA[
  var svgNS = "http://www.w3.org/2000/svg";
  function jump () {
    let dx = Math.random() * 100 - 50;
    let dy = Math.random() * 100 - 50;
    let elem = document.getElementById("message");
    elem.setAttribute("transform", "translate(" + dx + "," + dy + ")");
  }
  ]]></script>
</svg>
```

**Review by Ryuno-Ki — APPROVED** [review]
This already addresses the original issue.

**Pull request merged**